packages/mui-joy/src/Radio/Radio.tsx

```diff
@@ -302,6 +302,7 @@ const Radio = React.forwardRef(function Radio(inProps, ref) {
   const name = inProps.name || radioGroup?.name || nameProp;
   const disableIcon = inProps.disableIcon || radioGroup?.disableIcon || disableIconProp;
   const overlay = inProps.overlay || radioGroup?.overlay || overlayProp;
+  const isDisabled = inProps.disabled || formControl?.disabled || disabledProp;
```

Comment:
```suggestion
  const disabled = inProps.disabled || formControl?.disabled || disabledProp;
```

Let's keep `disabled` as a const name, we never use `is` as a prefix.

Reply:
there is already a variabled named `disabled`, hence went with `isDisabled`. if not for `isDisabled` what do you suggest?

https://github.com/mui/material-ui/blob/b7fea89bc232622546b6bc9675a818bfa95a8376/packages/mui-joy/src/Radio/Radio.tsx#L320